IndicoV2.Abstractions/Workflows/Models/Workflow.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using IndicoV2.StrawberryShake;
+
+namespace IndicoV2.Workflows.Models
+{
+    public class WorkflowSs : IWorkflow
+    {
+        private readonly IListWorkflows_Workflows_Workflows _ssWorkflow;
+        public WorkflowSs(IListWorkflows_Workflows_Workflows workflow) => _ssWorkflow = workflow;
+        public int Id => _ssWorkflow.Id ?? 0;
+        public bool ReviewEnabled => _ssWorkflow.ReviewEnabled ?? false;
```

Comment:
```suggestion
        public bool ReviewEnabled {get; set;} = false;
```

Reply:
https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties